packages/react/src/PricingOptions/PricingOptions.test.tsx

```diff
@@ -144,4 +144,77 @@ describe('PricingOptions.Item', () => {
 
     expect(testIdsInOrder).toEqual(expectedOrder)
   })
+
+  it('renders the PricingOptions.Heading with a h3 by default', () => {
+    const {getByRole} = render(
+      <PricingOptions.Item>
+        <PricingOptions.Heading>Mock heading</PricingOptions.Heading>
+      </PricingOptions.Item>,
+    )
+
+    const PricingOptionsItemEl = getByRole('heading')
+    expect(PricingOptionsItemEl.tagName).toBe('H3')
+  })
+
+  it.each<'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6'>(['h1', 'h2', 'h3', 'h4', 'h5', 'h6'])(
```

Comment:
Nice! 

You could even do `as const` if you want to remove the `'h1' | 'h2' | etc` too

```suggestion
  it.each(['h1', 'h2', 'h3', 'h4', 'h5', 'h6'] as const)(
```